db/migrate/20181126133409_add_external_service_id_to_exports.rb

```diff
@@ -0,0 +1,5 @@
+class AddExternalServiceIdToExports < ActiveRecord::Migration[5.2]
```

Comment:
records in the exports table need to have the external_system_id set (enforced in later migration).  This migration fixes existing data